cms/src/components/block/youtube-vimeo-block.json

```diff
@@ -1,33 +1,38 @@
 {
-    "collectionName": "components_block_youtube_vimeo_blocks",
-    "info": {
-        "displayName": "Youtube/Vimeo video",
-        "icon": "file-video",
-        "description": ""
+  "collectionName": "components_block_youtube_vimeo_blocks",
+  "info": {
+    "displayName": "Youtube/Vimeo video",
+    "icon": "file-video",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "url": {
+      "type": "string"
     },
-    "options": {},
-    "attributes": {
-        "url": {
-            "type": "string"
-        },
-        "provider": {
-            "type": "enumeration",
-            "enum": ["youtube", "vimeo", "facebook"]
-        },
-        "providerUid": {
-            "type": "string"
-        },
-        "width": {
-            "type": "integer"
-        },
-        "height": {
-            "type": "integer"
-        },
-        "title": {
-            "type": "string"
-        },
-        "thumbnailUrl": {
-            "type": "string"
-        }
+    "provider": {
+      "type": "enumeration",
+      "enum": [
+        "youtube",
+        "vimeo",
+        "facebook"
+      ]
+    },
+    "providerUid": {
+      "type": "string"
+    },
+    "width": {
+      "type": "integer"
+    },
+    "height": {
+      "type": "integer"
+    },
+    "title": {
+      "type": "string",
+      "required": true
+    },
+    "thumbnailUrl": {
+      "type": "string"
     }
+  }
 }
```

Comment:
@JakubVavra toto by som mozno vobec nemal tymto sposobom. Mame tam ten video field, ktory je urceny pre externe video a v podstate pokryva vacsinu z tychto veci ako je provider, url atd. To bol ucel, preco sme ten plugin / field vobec vytvorili, takze idealne by sme mali pouzit to a pripadne si ten plugin rozsirovat, takto mi to pride zbytocne.